### ISSUE TITLE
appdirs overwrites default paths

### ISSUE THREAD

**Issue by uetchy** [nils-werner/crestic#27]
<img width="703" alt="Screen Shot 2022-01-19 at 15 16 49" src="https://user-images.githubusercontent.com/431808/150074953-c84c0248-88e7-412f-93c4-07394ce9e367.png">

I prefer having crestic config at `/etc/crestic.cfg` or `/etc/crestic/crestic.cfg`, but here:

https://github.com/nils-werner/crestic/blob/051adf577db8ad20f00af9a42076ab568a521759/crestic.py#L15-L29

`appdirs` part overwrites the existing `paths` variable.

Can I suggest prepending the paths found from `appdirs` module to the existing paths?

Env: Arch Linux - `restic 0.12.1 compiled with go1.17.3 on linux/amd64` installed via `yay -S crestic`

**Comment by nils-werner** [comment]
Sounds reasonable. Do you want to open a PR?

**Comment by uetchy** [comment]
Very much. I’ll send a patch later.

**Comment by nils-werner** [comment]
Working with the config files logic myself I have realized that it works a lot differently from what I thought. Instead of reading just one file, `ConfigFile.read()` apparently reads all files and merges their content. I did not know that and designed everything incorrectly.

Because of this I will have to think about the config file logic a little bit more. Also I am considering dropping `appdirs` as it is barely maintained and just makes things a lot more complicated.
